docs/modules/ROOT/pages/explanation/decisions/source-of-users.adoc

```diff
@@ -0,0 +1,21 @@
+= Source of Users
+
+Problem::
+The source of the Users (User base) has to be somewhere well-defined.
+
+Relevant decisions::
+* xref:appuio-cloud:ROOT:explanation/decisions/arbitrary-usernames.adoc[User name choice on sign-up]
+* xref:appuio-cloud:ROOT:explanation/decisions/usernames.adoc[User object names in the OpenShift cluster]
+* xref:appuio-cloud:ROOT:explanation/decisions/stable-usernames.adoc[User name stability]
+* xref:appuio-cloud:ROOT:explanation/decisions/keycloak.adoc[Keycloak as IdP]
+
+Alternatives::
+* Store user base in xref:appuio-cloud:ROOT:explanation/decisions/keycloak.adoc[{idp}].
+
+Decision::
+The User base won't be in {idp}, it has to be in the resellers IdP, e.g. in VSHNs IdP.
+It has to be made available to {idp} using xref:how-to/keycloak-brokering.adoc[identity brokering].
+
+Rationale::
+The User base belongs to the reseller and not to APPUiO Cloud.
+The {idp} only stores a copy of the User and enhances it with APPUiO Cloud specific annotations.
```

Comment:
* I would avoid any mentions to VSHN at all
* If there is only 1 proposal, what is there to decide about? -> The decision should only name which alternative was chosen, not describe the proposal.
```suggestion
== Alternatives

[qanda]
Store user base in xref:appuio-cloud:ROOT:explanation/decisions/keycloak.adoc[{idp}]::
--> more details maybe?

The User base won't be in {idp}::
It will be in the reseller's resp. cluster-owner's IdP, made available to {idp} using xref:how-to/keycloak-brokering.adoc[identity brokering].

Decision::
The User base won't be in {idp}

Rationale::
The User base is provided by the reseller and not by {product}.
The {idp} only stores a copy of the User and adds {product} specific annotations.
```